{"name": "Тест входа в профиль курьера с неверным логином или паролем", "status": "passed", "description": "Тест проверяет появление ошибки в профиль курьера с неверным логином(1) и паролем(2)", "parameters": [{"name": "login", "value": "'invalidlogin'"}, {"name": "password", "value": "'valid_password'"}, {"name": "expected_status", "value": "404"}, {"name": "expected_message", "value": "'Учетная запись не найдена'"}], "start": 1721217085491, "stop": 1721217085493, "uuid": "e4e0dd01-09ec-465a-8c72-32ca076cc50d", "historyId": "c7541960465977f8782f5119f7823616", "testCaseId": "ec706951e0c7470079769d6d675d4cda", "fullName": "tests.test_login_courier.TestLoginCourier#test_login_invalid_credentials", "labels": [{"name": "feature", "value": "Тест входа в профиль курьера"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login_courier"}, {"name": "subSuite", "value": "TestLoginCourier"}, {"name": "host", "value": "DESKTOP-L3DLL49"}, {"name": "thread", "value": "12168-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login_courier"}]}